src/Sdk/WebApi/WebApi/LaunchHttpClient.cs

```diff
@@ -108,6 +109,21 @@ private static ActionDownloadInfo ToServerData(ActionDownloadInfoResponse action
             };
         }
 
+
+        private static ActionDownloadPackageDetails? ToServerData(ActionDownloadPackageDetailsResponse? actionDownloadPackageDetails)
+        {
+            if (actionDownloadPackageDetails == null)
+            {
+                return null;
+            }
+
+            return new ActionDownloadPackageDetails
+            {
+                Version = actionDownloadPackageDetails.Version,
+                ManifestDigest = actionDownloadPackageDetails.ManifestDigest
+            };
+        }
```

Comment:
Here's a more concise way to write this method:

```suggestion
        private static ActionDownloadPackageDetails? ToServerData(ActionDownloadPackageDetailsResponse? actionDownloadPackageDetails) =>
            actionDownloadPackageDetails == null ? null : new ActionDownloadPackageDetails
            {
                Version = actionDownloadPackageDetails.Version,
                ManifestDigest = actionDownloadPackageDetails.ManifestDigest
            };
```

Reply:
Fair enough 😄 I will probably leave it as is, personally I find that reads a little less clearly, and it is less consistent with existing code in that file: https://github.com/actions/runner/blob/ea2794d5d103e0ab3e8387e4ad1fd5613fd64086/src/Sdk/WebApi/WebApi/LaunchHttpClient.cs#L98-L110